src/pages/careers/_dept-layout-components/_hero.tsx

```diff
@@ -36,85 +27,22 @@ const StyledContainer = styled(Container)`
     max-width: 984px;
     flex-direction: column;
     align-items: flex-start;
-    height: 100%;
+    height: 660px;
```

Comment:
Hi @niloofar-deriv , fixing this height for desktop might affect the mobile devices

Reply:
I defined another height for mobile 
```
const StyledContainer = styled(Container)`
    max-width: 984px;
    flex-direction: column;
    align-items: flex-start;
    height: 660px;

    @media ${device.tablet} {
        height: 560px;
        padding: 0 2rem;
    }
`
```